oxidation/crates/parsec_serialization_format/src/parser/mod.rs

```diff
@@ -3,16 +3,16 @@
 mod data;
 mod field;
 mod protocol;
-mod schema;
+#[allow(dead_code)]
 mod serde_attr;
+#[allow(dead_code)]
 mod serializing;
 pub(crate) mod utils;
 mod variant;
 
 pub(crate) use data::*;
 pub(crate) use field::*;
 pub(crate) use protocol::*;
-pub(crate) use schema::*;
 pub(crate) use serde_attr::*;
-pub(crate) use serializing::*;
+// pub(crate) use serializing::*;
```

Comment:
dead code ?

Reply:
so should be removed ;-)

Reply:
I will keep it, I think it's a better approach if we can specify how it's serialized/deserialized with a #[derive]

Reply:
YAGNI, just remove it now and we will re-added it when it's needed

Reply:
Done :+1: